src/main/java/de/retest/recheck/review/workers/SaveShouldIgnoreWorker.java

```diff
@@ -21,13 +24,16 @@ public SaveShouldIgnoreWorker( final GlobalIgnoreApplier applier ) {
 	}
 
 	public void save() throws IOException {
-		final Optional<Path> path = RecheckIgnoreUtil.getIgnoreFile();
+		final Optional<Path> ignoreFile = RecheckIgnoreUtil.getIgnoreFile();
 		final PersistableGlobalIgnoreApplier persist = applier.persist();
 
-		final Stream<String> save = Loaders.save( persist.getIgnores().stream() );
+		// Filter JSShouldIgnore because that would create unnecessary file content.
+		final Stream<ShouldIgnore> ignores = persist.getIgnores().stream() //
+				.filter( ((Predicate<ShouldIgnore>) JSShouldIgnoreImpl.class::isInstance).negate() );
```

Comment:
Which is more readable?
```suggestion
				.filter( (ignore -> !(ignore instanceof JSShouldIgnoreImpl) );
```